systest/backup/nfs-backup/docker-compose.yml

```diff
@@ -0,0 +1,377 @@
+
+version: "3.5"
+services:
+#HA backup cluster
+  alpha1:
+    image: dgraph-nfs-client
+    working_dir: /data/alpha1
+    links:
+      - "nfs:nfs"
+    privileged: true
+    labels:
+      cluster: test
+    ports:
+    - "8080"
+    - "9080"
+    volumes:
+    - type: bind
+      source: $GOPATH/bin
+      target: /gobin
+      read_only: true
+
+    - type: bind
```

Comment:
we actually can live without TLS in these tests. That will reduce the number of files you have added significantly. Though, it may require improving some of the test utilities.